src/components/input/_index.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../scss";
+
```

Comment:
I don't see any hooks for providing the ability to add styles for the label.

This doesn't quite seem quite right - as the comment is called `input` but we also need to expose the ability to style the `<input />` too

```css
.c-input {
	@include scss.component-properties("input");
}
```

Reply:
what? the inputs are styled via: 

```scss
.c-input__input--error {
	@include scss.component-properties("input-error");
}

.c-input__input--success {
	@include scss.component-properties("input-success");
}

.c-input__input--warning {
	@include scss.component-properties("input-warning");
}

// added placeholder for default status
// https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder
.c-input__input--default {
	@include scss.component-with-states("input-default", [placeholder]);
}

```

Reply:
will add a way to style the label -- good call

Reply:
addressed and updated